crates/bevy_render/src/texture/texture_attachment.rs

```diff
@@ -120,3 +120,41 @@ impl DepthAttachment {
         }
     }
 }
+
+/// A wrapper for a [`TextureView`] that is used as a [`RenderPassColorAttachment`] for a view
+/// targets final output texture.
+#[derive(Clone)]
+pub struct OutputColorAttachment {
+    pub view: TextureView,
+    pub format: TextureFormat,
+    is_first_call: Arc<AtomicBool>,
```

Comment:
This needs documenting. The object has nothing to do with a function call, so it's impossible to understand what the "first call" refers to. I'd even suggest entirely avoiding "call" here, something like `is_first_use` rather?

Reply:
I agree the naming isn't ideal but I'd rather see it renamed in another PR that changes it for every place this pattern is used not just here.

Reply:
Yeah, this is just following the existing pattern in the file. Agree it's not totally clear (ha!).